Update the activity app (#78)
owncloud/docs-webui: modules/classic_ui/pages/apps/activity.adoc

image::apps/activity/activity-menu.png[View activity in ownCloud's WebUI.]

> By default, you see all activity related to your files and folders. However, by using the left-hand navigation menu, you can filter activity by:

Reviewer: ```suggestion
By default, you see all activity related to your files and folders. However, by using the left-hand navigation menu, you can filter activities by:
```

== Configuring the Activity App

> To configure your Activity preferences navigate to menu:Settings[Personal > General > Activity]. You can configure notifications for when:

Reviewer: ```suggestion
To configure your Activity preferences, navigate to menu:Settings[Personal > General > Activity]. You can configure notifications for events like:
```

=== Configuring the Email Notification Interval

In addition to enabling and disabling email notifications, bulk email notifications can be configured to be sent out: _As soon as possible_ (during the next cron execution), _Hourly_, _Daily_, and _Weekly_. To do so, pick the interval in the "_Send emails:_" drop-down field at the bottom of the Activity configuration panel.

== Time Base

> * An admin can define a default date for shares to expire you can overwrite if needed. Whenever a default date is enabled and set for expiring shares (e.g. after 7 days), the time base for default dates is the time of the ownCloud server, not the time of the client accessing ownCloud Server.

Reviewer 2: ```suggestion
* An admin can define a default date for shares to expire, which you can change if needed. Whenever a date is set for expiring shares (e.g. after 7 days), the share expires at the end of that day. The time base is the time of the ownCloud server, not the time of the client accessing the ownCloud Server.
```

> * An admin can define a default date for shares to expire, which you can change if needed. Whenever a date is set for expiring shares (e.g. after 7 days), the share expires at the end of that day. The time base is the time of the ownCloud server, not the time of the client accessing the ownCloud Server.

Reviewer: ```suggestion
* An admin can define a default date for shares to expire, which you can change if needed. Whenever a date is set for expiring shares (e.g. after 7 days), the share expires at the end of that day. The time base is the time of the ownCloud server, not the time of the client accessing the ownCloud server.
```

> * When a client recieves an expiry notifications for a share, the notification is by the end of the day based on the time of the ownCloud server and not the time of the client accessing ownCloud Server.

Reviewer 2: ```suggestion
* When a client receives an expiry notification for a share, the expiry is effective at the end of the day based on the time of the ownCloud server and not the time of the client accessing ownCloud Server.
```

> * When a client receives an expiry notification for a share, the expiry is effective at the end of the day based on the time of the ownCloud server and not the time of the client accessing ownCloud Server.

Reviewer: ```suggestion
* When a client receives an expiry notification for a share, the expiry is effective at the end of the day based on the time of the ownCloud server and not the time of the client accessing ownCloud server.
```
